### PR TITLE
Update 'Getting started' doc and related docs

### PR DESCRIPTION
**Pull request by jandonyada**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -209,6 +209,7 @@
         * [Custom Workbot vs Legacy slash commands](workbot/custom-workbot-vs-legacy-slash-commands.md)
 * [Workbot for MS Teams](/workbot-for-teams/workbot.md)
   * [Using Workbot for MS Teams](/workbot-for-teams/using-workbot-for-teams.md)
+  * [Getting started](/workbot-for-teams/getting-started.md#getting-started)
```
Comment (eeshansim) [inline SUMMARY.md]: Do we really need an anchor here?

File: workbot-for-teams/getting-started.md [hunk 2]
```diff
@@ -40,6 +40,27 @@ Workbot commands allow you to trigger recipes from Teams. These recipes can then
 
 >**For more info, go to [Workbot commands](/workbot-for-teams/workbot-triggers.md)**.
 
+## Getting started
+### Requirements
+To use Workbot for Teams, a [**Microsoft work or school account**](https://support.office.com/en-ie/article/what-account-do-i-use-with-office-and-why-do-i-need-one-914e6610-2763-47ac-ab36-602a81068235?ui=en-US&rs=en-IE&ad=IE#bkmk_msavsworkschool) is required. These accounts are created by your organization’s administrator to enable a member of your organization access to all Microsoft cloud services such as Microsoft Azure, Office 365, Microsoft Teams, and more.
+
+>**Microsoft accounts (typically created for personal use) are not supported.**
```
Comment (eeshansim) [inline workbot-for-teams/getting-started.md]: Use space after `>` for block quotes

File: SUMMARY.md (hunk 1 above)
Comment (jandonyada) [reply SUMMARY.md]: Yea, so this doc can be opened from an iFrame from a freshly installed Workbot in the Teams client. The top part of the doc introduces the user to Workato, recipe basics and Workbot.

This anchor skips all that and brings the user to the setup, permissions and requirements.

Comment (eeshansim) [reply SUMMARY.md]: Feels very weird that a getting started page needs a special section for getting started

**Pull request merged**